test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -166,6 +167,38 @@ def run_model_test_with_external_data(self, model, input, rtol=0.001, atol=1e-7,
                 input_copy = copy.deepcopy(input)
                 ort_test_with_input(ort_sess, input_copy, output, rtol, atol)
 
+    def run_ort(self, model, batch_size=2, state_dict=None,
```

Comment:
We have run_model_test and ort_test_with_input methods that have 90% code similarity here.
You can separate the ort_test_with_input into two steps:
1- Run ORT model
2- Compare outputs

And reuse the code.
No need to add a new function.